packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -303,6 +316,7 @@ export class ElectronMainApplication {
         return {
             show: false,
             title: this.config.applicationName,
+            backgroundColor: nativeTheme.shouldUseDarkColors ? '#1E1E1E' : '#FFFFFF',
```

Comment:
A similar dispatch is already at:
 https://github.com/eclipse-theia/theia/blob/f038673dc228485b34e8762686525d8e5f258afb/packages/core/src/browser/preload/theme-preload-contribution.ts#L28

Could you single-source them somewhere here? Thanks
 https://github.com/eclipse-theia/theia/blob/f038673dc228485b34e8762686525d8e5f258afb/dev-packages/application-package/src/application-props.ts#L48-L63